backend/.env.example

```
@@ -3,3 +3,13 @@ DB_PORT=5432
 POSTGRES_USER=postgres
 POSTGRES_PASSWORD=change_me
 POSTGRES_DB=converge_db
+
+JWT_SECRET=change_me_to_a_long_random_string
+JWT_TTL=24h
+
+ADMIN_USERNAME=admin
+ADMIN_PASSWORD=change_me
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ADMIN_PASSWORD key should go before the ADMIN_USERNAME key

(UnorderedKey)